kafka-connect-s3/src/test/java/io/confluent/connect/s3/TopicPartitionWriterTest.java

```diff
@@ -663,12 +875,12 @@ public void testNoFilesWrittenWithoutCommit() throws Exception {
     Partitioner<FieldSchema> partitioner = new DefaultPartitioner<>();
     partitioner.configure(parsedConfig);
     TopicPartitionWriter topicPartitionWriter = new TopicPartitionWriter(
-        TOPIC_PARTITION, storage, writerProvider, partitioner,  connectorConfig, context);
+        TOPIC_PARTITION, storage, writerProvider, partitioner, connectorConfig, context);
 
     String key = "key";
     Schema schema = createSchema();
     List<Struct> records = createRecordBatches(schema, 3, 3);
-
+TimeUnit.SECONDS.toMillis(20);
```

Comment:
What is this line attempting to do?

Reply:
Just some extra math to keep the CPU on its toes!

removed -- it was a stray copy-pasta